src/components/details/styles.jsx

```diff
@@ -0,0 +1,41 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/FeNixCrew/MiEntradita-frontend/pullRequest?prid=8421134)
```suggestion
import { makeStyles } from "@material-ui/core/styles";
```